docs/terms.md

```diff
@@ -0,0 +1,31 @@
+# Terms and Conditions
+
+Clangd remote index service is run on a best-effort basis by C++ tools team at
+Google. Google donated the source code and cloud resources to the LLVM project,
+but is not an official Google product. We also monitor the service on a
+best-effort basis and will do our best to deal with arising problems (such as
+server not being responsive) within several working days. We understand that
+productivity of remote index users would be affected in case of the server
```

Comment:
that's a reasonable promise indeed. but 5% of a week is ~8 hours, and there are weekends. maybe change the promise to "it won't stay down for more than 1 workday"? also if possible i would still avoid that in TC, is it certain that we need this in the TC?

Reply:
I think this was something we were asked about so may be a good thing to have but I'm not certain.